examples/user_guide/Exporting_and_Archiving.ipynb

```diff
@@ -155,7 +155,7 @@
    "source": [
     "%%opts Contours (linewidth=1.3) Image (cmap=\"gray\")\n",
     "cs = contours(penguins[:,:,'R'], levels=[0.10,0.80])\n",
-    "cs"
+    "penguins[:, :, 'R'] * cs"
```

Comment:
The operation no longer just adds contours on top of the input and return it? That's fine as long as there is a switch to restore the old behavior. I also suspect we may have made this change a while back.

Reply:
I see you changed the default - see comment below.

Reply:
We would at least need to document this change of behavior in the changelog/release notes.

Reply:
Right, may start requiring committers adding backwards compatibility changes to the CHANGELOG when they are made so we don't forget about them. Will do that here.